dotnet-desktop-guide/net/winforms/data/how-to-work-with-data-grid.md

```diff
@@ -97,9 +97,8 @@ To test the code example, perform the following steps:
 
 ## See also
 
-- [Data Sources Supported by Windows Forms](/dotnet/desktop/winforms/data-sources-supported-by-windows-forms?view=netframeworkdesktop-4.8&preserve-view=true)
-- [Change Notification in Windows Forms Data Binding](/dotnet/desktop/winforms/change-notification-in-windows-forms-data-binding?view=netframeworkdesktop-4.8&preserve-view=true)
-- [Windows Forms Data Binding](/dotnet/desktop/winforms/windows-forms-data-binding?view=netframeworkdesktop-4.8&preserve-view=true)
-- [How to Ensure Multiple Controls Bound to the Same Data Source Remain Synchronized](/dotnet/desktop/winforms/multiple-controls-bound-to-data-source-synchronized?view=netframeworkdesktop-4.8&preserve-view=true)
+- [Data binding overview](overview.md)
+- [Data sources supported by Windows forms](overview.md#data-sources-supported-by-windows-forms)
```

Comment:
Every other use of "Windows Forms" capitalizes "Forms". Should this one as well?

```suggestion
- [Data sources supported by Windows Forms](overview.md#data-sources-supported-by-windows-forms)
```